XYZDrivers/src/main/java/com/xyzdrivers/controllers/AdminController.java

```diff
@@ -5,19 +5,33 @@
  */
 package com.xyzdrivers.controllers;
 
-import java.io.IOException;
-import java.io.PrintWriter;
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import com.xyzdrivers.models.*;
+
+import java.io.*;
+import java.sql.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
 
 /**
  *
  * @author arthur
  */
 public class AdminController extends HttpServlet {
 
+    public List<Object[]> getMembersList(jdbcDriver JDBC)
```

Comment:
This should be moved to a service (e.g. MembersService) so we don't have lots of code in the controller.